pylsp/plugins/autopep8_format.py

```diff
@@ -13,13 +13,13 @@
 
 
 @hookimpl(tryfirst=True)  # Prefer autopep8 over YAPF
-def pylsp_format_document(config, document):
+def pylsp_format_document(config, document, _options=None):
```

Comment:
```suggestion
def pylsp_format_document(config, document, options=None):
```

Let's leave this parameter without an underscore, even if it's unused.